### PR TITLE
Release 0.14.0

### PR DESCRIPTION
**Pull request by PVince81**
Closes https://github.com/owncloud/files_antivirus/issues/278

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,15 +3,28 @@
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+## [0.14.0] - 2018-11.30
```
Comment (davitol) [inline CHANGELOG.md]: 2018-11.30  it has a point instead of a -

File: CHANGELOG.md [hunk 2]
```diff
@@ -258,6 +272,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Initial implementation
 
-[Unreleased]: https://github.com/owncloud/files_antivirus/compare/v0.13.0...HEAD
-[0.13.0]: https://github.com/owncloud/files_antivirus/compare/v0.12.0...release/0.13.0
-[0.12.0]: https://github.com/owncloud/files_antivirus/compare/v0.11.2...release/0.12.0
\ No newline at end of file
+[Unreleased]: https://github.com/owncloud/files_antivirus/compare/v0.13.0...master
```
Comment (phil-davis) [inline CHANGELOG.md]: should be ``v0.14.0...master`` ?

File: CHANGELOG.md (hunk 1 above)
Comment (PVince81) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (PVince81) [reply CHANGELOG.md]: fixed

**Comment by PVince81** [comment]
restarted cancelled build

**Pull request merged**